src/Common/src/CoreLib/System/Text/ValueStringBuilder.cs

```diff
@@ -185,7 +197,7 @@ private void Grow(int requiredAdditionalCapacity)
         }
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        private void Clear()
+        public void Clear()
```

Comment:
Nit: It may be better to call it `Dispose` now that it is public.

Reply:
I assume going further and adding IDisposable would be too heavyweight for this -- especially since "leaking" is not a big deal.

Reply:
Right. Also, interfaces do not really make sense for ref types - you cannot cast the ref type to the interface.

Reply:
A tangent: I haven't tried this, but I imagine a ref struct with an interface might have some limited use in the context of constrained generics. That is, if you had

```csharp
ref struct Foo : IBar {}
void Baz<T>(T baz) where T : IBar
```

you should be able to write

```csharp
Baz(new Foo());
```

Reply:
Although: I guess you're probably not allowed to use a ref struct as a type argument, because you don't know whether the method will use the `T` in a way that you can't use a ref struct (eg casting it to `object`).